### PR TITLE
COP-1164 Dashboard panel design changes

### PR DESCRIPTION
**Pull request by jennikate**
Using [https://cop.prototype.cop.homeoffice.gov.uk/blue/sprint-34/dashboard-v0-3?nginxId=49874594-bc95-429b-c256-eac2372ee0cb](https://cop.prototype.cop.homeoffice.gov.uk/blue/sprint-34/dashboard-v0-3?nginxId=49874594-bc95-429b-c256-eac2372ee0cb) as a guide

**Notes**

- keep desktop page width as is (1400) do not reduce to match prototype
- wording not yet updated, only style

**Changes for this PR**

- Use Home Office color for titles
- Titles of panel should be the link and behave like links
- Text below should be gov-body small text (16px on wide, 14px on mobile)
- Counts for tasks and messages should be maintained

### CODE REVIEW

File: app/pages/dashboard/components/DashboardPage.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import DashboardTitle from './DashboardTitle';
 import DashboardPanel from './DashboardPanel';
 import * as actions from '../../../core/shift/actions';
 import * as errorActions from '../../../core/error/actions';
-import "./Dashboard.scss";
+import "../../../../public/styles/homeoffice.scss"
```
Comment (p3dr0migue1) [inline app/pages/dashboard/components/DashboardPage.jsx:10]: :+1:

File: app/pages/dashboard/components/TaskCountPanel.jsx [hunk 2]
```diff
@@ -81,64 +81,40 @@ export class TaskCountPanel extends React.Component {
   render() {
     const { taskCounts, isFetchingTaskCounts } = this.props;
     return (
-      <div>
-        <li className="__card govuk-grid-column-one-third" id="yourTasksPanel">
+      <React.Fragment>
```
Comment (p3dr0migue1) [inline app/pages/dashboard/components/TaskCountPanel.jsx:84]: Good update

File: public/styles/homeoffice.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+// .home-top {
+
+// }
+
+// .home-top__title {
+
+// }
+
+// .home-home-top__links {
```
Comment (p3dr0migue1) [inline public/styles/homeoffice.scss:11]: Is this commented out code for the next story? For the future lets try and avoid committing code that has been commented out.

Comment (jennikate) [reply public/styles/homeoffice.scss:11]: 👍 (it is for next story!)

**Pull request merged**